modules/gateway/consts.go

```diff
@@ -8,9 +8,14 @@ import (
 
 const (
 	// handshakeUpgradeVersion is the version where the gateway handshake RPC
-	// was altered to include adiitional information transfer.
+	// was altered to include additional information transfer.
 	handshakeUpgradeVersion = "1.0.0"
 
+	// sessionHandshakeUpgradeVersion is the version where the gateway handshake RPC
+	// was altered to include the ID of the genesis block, the gateway's unique ID,
+	// and whether a connection is desired.
+	sessionHandshakeUpgradeVersion = "1.1.2"
```

Comment:
that's the plan, yeah.
Changing this string might break your tests, not sure. In that case I suppose we should just push a commit to master that updates the version number to 1.2.0.